### DISCUSSION TITLE
Passing current timestamp as default value

### DISCUSSION THREAD

**Discussion by niveditabaliga-payroc** [airbytehq/airbyte#40597] [Connector Builder]
Hello Fellow Airbyters,
I built a custom API connector to get NiceCXOne data. The call requires that we pass a start and end date. When I deploy this connector, I want the start date to be yesterday and end date to be today. The option to enter a "Default Value" only gives a calendar, as in screenshot below.
<img width="534" alt="Screen Shot 2024-06-27 at 10 39 43 AM" src="https://github.com/airbytehq/airbyte/assets/159464153/65e00c35-6c75-4cd0-8b26-7a1944e002f5">

How do I pass default values yesterday, today (or current_timestamp - 1 and current_timestamp)?

**Comment by xhensiladoda** [comment]
I would be very interested in this feature as well.
Did you manage to succeed @niveditabaliga-payroc ?

**Comment by niveditabaliga-payroc** [comment]
@xhensiladoda I was able to implement it. I passed the parameter as a "Request Body Parameter" and set the value to be
`{{ default(format_datetime(now_utc(), '%Y-%m-%dT%H:%M:%S%z') | replace('+0000', '+00:00')) }}`
Note: Some of the date cleansing is specific to my API call and may differ for yours.
<img width="981" alt="Screenshot 2025-05-27 at 10 06 42 AM" src="https://github.com/user-attachments/assets/b2e255a5-f316-445e-a658-f941ac38b52d" />

**Reply by xhensiladoda** [reply]
Thanks a lot @niveditabaliga-payroc for your reply.
Indeed, I did the same workaround as you, since apparently from the Input parameters it cannot be done.
Here is also my solution if in the future someone else needs it.
![image](https://github.com/user-attachments/assets/5c3521d9-14f6-4c9e-a697-38ceb1d4885d)
